public/user_img.html

```diff
@@ -32,7 +32,7 @@
     <!-- Navigation -->
     <nav class="navbar navbar-expand-lg bg-secondary fixed-top text-uppercase" id="mainNav">
       <div class="container">
-        <a class="navbar-brand js-scroll-trigger" href="#page-top">Fam</a>
+        <a href="index.html" class="navbar-brand js-scroll-trigger" href="#page-top">Fam</a>
```

Comment:
not sure about multiple href's here, personally I would have one href pointing to "/"

Reply:
Also, see https://github.com/Brechy/fam/issues/47